java/example_code/cloudmap/src/main/java/aws/example/cloudmap/CreateService.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Author : Rajib Deb
+ * 
+ * This program creates a service in the given namespace in AWS Cloud Map
```

Comment:
Please move this sentence to the metadata for sourcedescription?

Reply:
This is done